workflow-templates/job.push.yml

```
@@ -0,0 +1,68 @@
+name: Update Image Tag in Config Repository
+
+on:
+  workflow_call:
+    inputs:
+      image_tag:
+        required: true
+        type: string
+      aws_region:
+        required: true
+        type: string
+      environment:
+        required: true
+        type: string
+      service:
+        required: true
+        type: string
+      namespace:
+        required: false
+        type: string
+        default: "liquid"
+      config_repo:
+        required: false
+        type: string
+        default: "gainsway/cluster-workloads"
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+    env:
+      values_path: "environments/liquid/${{ inputs.aws_region }}/${{ inputs.environment }}/values.yaml"
+    permissions:
+      contents: read
+      packages: read
+
+    steps:
+      - name: Display inputs
+        run: |
+          echo "Image Tag: ${{ inputs.image_tag }}"
+          echo "aws_region: ${{ inputs.aws_region }}"
+          echo "environment: ${{ inputs.environment }}"
+          echo "namespace": ${{ inputs.namespace }}
+          echo "service: ${{ inputs.service }}"
+
+      - name: Checkout Repo
+        uses: actions/checkout@v4
```

We need to use a token otherwise the Update Image step will fail.
Of course we would also need the secret as param of the GH trigger

```suggestion
      - name: Generate token
        id: generate_token
        uses: actions/create-github-app-token@v1
        with:
          app_id: ${{ secrets.APP_ID }}
          private_key: ${{ secrets.APP_KEY }}

      - name: Checkout Repo
        uses: actions/checkout@v4
        with:
          token: ${{ steps.generate_token.outputs.token }}
```